test-result-summary-client/src/Build/GitNewIssue.jsx

```diff
@@ -54,6 +59,26 @@ export default class GitNewissue extends Component {
                 }
             }
 
+            let failureJ9JavaVersion, successJ9JavaVersion;
+
+            // We will only show git diff if Java Version is OpenJ9 VM
+            if (!isCurrentFailure && successBeforeFailure && firstSeenFailure.javaVersion.includes("OpenJ9 VM") && successBeforeFailure.javaVersion.includes("OpenJ9 VM")) {
+                // first seen failure is not current failure
+                failureJ9JavaVersion = firstSeenFailure.javaVersion;
+                successJ9JavaVersion = successBeforeFailure.javaVersion;
+            } else if (isCurrentFailure && javaVersion.includes("OpenJ9 VM") && successBeforeFailure.javaVersion.includes("OpenJ9 VM")) {
+                // first seen failure is current result
+                failureJ9JavaVersion = javaVersion;
+                successJ9JavaVersion = successBeforeFailure.javaVersion;
+            }
+
+            if (failureJ9JavaVersion && successJ9JavaVersion) {
+                if (getGitDiffLinks(failureJ9JavaVersion, successJ9JavaVersion)) {
+                    const { gitDiffOpenJ9, gitDiffOMR } = getGitDiffLinks(failureJ9JavaVersion, successJ9JavaVersion);
+                    gitDiffLinks = `Git Diff for OpenJ9: ${gitDiffOpenJ9}\nGit Diff for OMR: ${gitDiffOMR}`;
```

Comment:
Due to the limited length, our msg needs to be concise. Since we have the title, I think we can remove `Git Diff for XXX: `